unit-tests/func/func-common.h

```diff
@@ -20,6 +22,45 @@ inline rs2::device_list find_devices_by_product_line_or_exit( int product )
     return devices_list;
 }
 
+// This function returns the first device found that it's name contains the given string,
+// if there is no such device it will exit the test.
+// Can get as input a full device name or short like "L515/D435..."
+inline rs2::device find_device_by_name_or_exit( const std::string & dev_name )
```

Comment:
If we want to return only a single device, then the convention already set by the other (py) function is to use `first` in the name: `find_first_device_by_name_or_exit`.

However, we have another convention already set: return all devices matching, but I'm OK going forward with the single device until we figure out what the infrastructure with acroname would look like.

Reply:
Agreed , I renamed it to "find_first_device_by_name_or_exit",